pkg/api/requests/namespace.go

```diff
@@ -34,7 +34,10 @@ type NamespaceDelete struct {
 // NamespaceEdit is the structure to represent the request data for edit namespace endpoint.
 type NamespaceEdit struct {
 	TenantParam
-	Name string `json:"name"  validate:"required,hostname_rfc1123,excludes=."`
+	Name     string `json:"name"  validate:"omitempty,hostname_rfc1123,excludes=."`
```

Comment:
Is `omitempty` a valid `validate` tag?

Reply:
Yes, you read about [here](https://pkg.go.dev/github.com/go-playground/validator/v10#hdr-Omit_Empty).